Rakefile

```diff
@@ -17,7 +17,7 @@ task :default => [:test]
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = 'spec/{unit}/**/*.rb'
+  t.pattern = 'spec/{classes,defines,unit,functions,hosts,integration}/**/*_spec.rb'
```

Comment:
I was going to ask if this was necessary (the Rakefile), but went and looked at other modules Rakefiles and determined we were doing different things with different modules. Probably just a neglect thing, need to figure out the best thing and do that.